.travis.yml

```diff
@@ -3,6 +3,8 @@ sudo: required
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
 script: bash -ex .travis-opam.sh
 env:
+  global:
+    - PINS="zmq:.zmq-async:."
```

Comment:
I think we should test both libraries, so it would become 
```yaml
language: c
sudo: required
install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
script: bash -ex .travis-opam.sh
env:
  global:
    - TESTS=true
  matrix;
    - OCAML_VERSION=4.03 PACKAGE="zmq"       PINS="zmq:." 
    - OCAML_VERSION=4.03 PACKAGE="zmq-async" PINS="zmq:. zmq-async:."
    - OCAML_VERSION=4.04 PACKAGE="zmq"       PINS="zmq:." 
    - OCAML_VERSION=4.04 PACKAGE="zmq-async" PINS="zmq:. zmq-async:."
    - OCAML_VERSION=4.05 PACKAGE="zmq"       PINS="zmq:." 
    - OCAML_VERSION=4.05 PACKAGE="zmq-async" PINS="zmq:. zmq-async:."
    - OCAML_VERSION=4.06 PACKAGE="zmq"       PINS="zmq:." 
    - OCAML_VERSION=4.06 PACKAGE="zmq-async" PINS="zmq:. zmq-async:."
os:
  - linux
```

Reply:
As there are two opam files, the travis tests cannot determine the package, hence it needs to be defined explicitly.
In the case where we want to test zmq-async, we need to pin the zmq package so it can be pulled in when installing.